docs/contribute/style-guide.md

```diff
@@ -152,7 +150,7 @@ elements](https://learn.microsoft.com/en-us/style-guide/text-formatting/):
 
 ❌ *[Click here](https://discord.gg/hyperledger) for Besu support.*
 
-✅ *If you have questions about Besu for public networks, ask on the **besu** channel on
+✅ *If you have questions about Besu for public networks, ask on the **Besu** channel on
 [Hyperledger Discord](https://discord.gg/hyperledger).*
```

Comment:
```suggestion
✅ *If you have questions about Besu for public networks, ask on the **#besu** channel on
[LFDT Discord](https://discord.gg/hyperledger).*
```

This is lowercase "besu" because that's how it appears as a Discord channel. Is clearer to include the # sign? Either way is fine for me though. Also Hyperledger name change.